platform-api/che-core-api-builder/src/main/java/org/eclipse/che/api/builder/BuilderSelectionStrategy.java

```diff
@@ -22,4 +23,8 @@
  */
 public interface BuilderSelectionStrategy {
     RemoteBuilder select(List<RemoteBuilder> slaveBuilders);
+
+    default RemoteBuilder select(List<RemoteBuilder> slaveBuilders, BaseBuilderRequest request){
+        return select(slaveBuilders);
+    }
```

Comment:
The default implementation for the added method was for automatically support other strategy implementations, not from Che code. so other who took the che code and created their own strategy implementation will not break. if it's not a real case , then we can just update the single method of the interface and the current two implementations